src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3255,12 +3272,25 @@ void os::large_page_init() {
     return;
   }
 
-  _large_page_size = large_page_init_decide_size();
+  std::this_thread::sleep_for(std::chrono::seconds(30));
```

Comment:
Is this and the additional #includes above test code?

Reply:
I will revert this

Reply:
Reverted my last commit that had introduce it.